serviceWorkerConfig.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * Configuration for the serviceworker.
+ */
+
+window.serviceWorkerConfig = {
+	// // A function that is run to decide whether to register the SW or not.
+	// You could for instance make this return a value based on whether you actually need to be cross origin isolated or not.
+	shouldRegister: () => true,
+	// If this function returns true, any existing service worker will be deregistered (and nothing else will happen).
+	shouldDeregister: () => false,
+	// A function that is run to decide whether to use "Cross-Origin-Embedder-Policy: credentialless" or not.
+	// See https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Cross-Origin-Embedder-Policy#browser_compatibility
+	coepCredentialless: () => !(navigator.userAgent.indexOf("CriOS") > -1 || !window.chrome),
```

Comment:
What is the rationale here? WebVM cannot work without COI at all

Reply:
The rationale was that coep credentialless also enables COI in browsers that support it, and has the benefit of not exposing credentials. I was not sure whether this was also applicable to us, but I saw no reason to not add it as an option.

Reply:
It seem to have no usefulness to us, and it's just one more thing to maintain in time. In general only keep what is strictly necessary